packages/protocol/src/action/request/Write.ts

```diff
@@ -41,13 +52,25 @@ export function Write(optionsOrData: Write.Options | Write.Attribute, ...data: W
     } else {
         options = optionsOrData;
     }
-    const { writes: writeRequests = [] } = options;
+    const { writes: writeRequests = [], timed, timeout, chunkLists } = options;
 
-    const result: Write = {
-        timedRequest: !!options.timed || !!options.timeout,
+    const result = {
+        timedRequest: !!timed || !!timeout,
+        timeout,
         writeRequests,
-        interactionModelRevision: options.interactionModelRevision ?? FALLBACK_INTERACTIONMODEL_REVISION,
-    };
+        moreChunkedMessages: false,
+        interactionModelRevision: options.interactionModelRevision ?? Specification.INTERACTION_MODEL_REVISION,
+
+        [Diagnostic.value]: () =>
+            Diagnostic.list(
+                data.map(entry => {
+                    const { version, value } = entry;
+                    return `${resolvePathForSpecifier(entry)} = ${Diagnostic.json(
```

Comment:
So r.e. comment on `resolvePathForSpecifier`, use `ExpandedPath` instead in places like this